### DISCUSSION TITLE
strange startup error

### DISCUSSION THREAD

**Discussion by ghost** [ntsc-rs/ntsc-rs#98] [Q&A]
Do any of you know what this error is?
[2024-05-21T10:31:08Z ERROR eframe::native::run] Exiting because of error: egui_glow: OpenGL: egui_glow requires opengl 2.0+.  during event Resumed
Error: OpenGL(PainterError("egui_glow requires opengl 2.0+. "))

I tried to run on WS 2022

**Comment by valadaptive** [comment]
What graphics card do you have, and have you installed the drivers for it?

**Comment by ghost** [comment]
I used a VM and WS 2022. Hypervisor Hyper-V

**Comment by valadaptive** [comment]
I'm not sure what virtualization software you're using, but you should enable 3D acceleration if it's available. If you can't do so, then ntsc-rs will not run.

**Comment by valadaptive** [comment]
Seems like this is resolved now.
